src/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -189,17 +187,26 @@ public enum WindowsBuiltInRole
     }
     public partial class WindowsIdentity : System.Security.Claims.ClaimsIdentity, System.IDisposable
     {
+        public new const string DefaultIssuer = "AD AUTHORITY";
```

Comment:
I'd be concerned that a consumer of this const (which is described on MSDN) when moving to CoreFX would find themselves suddenly (and silently) picking up ClaimsIdentity::DefaultIssuer ("LOCAL AUTHORITY") instead of WindowsIdentity::DefaultIssuer ("AD AUTHORITY").

Not that I really know what that's used for, but it does seem like not having the const in the contract here is effectively as breaking of a change as replacing its value.